release-notes/aws/v9.0.0.md

```diff
@@ -23,7 +23,7 @@
   - The default value of configmap setting `proxy-add-original-uri-header` is now `false`.
   - When the setting `proxy-add-original-uri-header` is `true`, the ingress controller adds a new header `X-Original-Uri` with the value of NGINX variable `$request_uri`. In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the `Ingress` `serviceName`, like issue https://github.com/kubernetes/ingress-nginx/issues/4593.
 
-### coredns v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
+### coreDNS v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
```

Comment:
How about putting those links on the bottom of the file like here https://raw.githubusercontent.com/giantswarm/architect-orb/master/CHANGELOG.md ?

Reply:
There could be multiple links e.g. `[0.5.0]` for multiple projects here - does this still work then?